docs/quickstart.rst

```diff
@@ -20,7 +20,7 @@ We strongly recommended to create a virtual environment for any new project:
     pip install --upgrade pip wheel setuptools
 
 
-Then clone the GT4Py repository and install the local copy or install it directly from GitHub with `pip`.
+Then clone the GT4Py repository and install the local copy or install it directly from pypi.org.
```

Comment:
```suggestion
Then clone the GT4Py repository and install the local copy or install it directly from PyPI: `pip install gt4py`
```

Reply:
Ok you say it below, but the section mentions a bunch of things and only shows one. maybe at least make it clear that pip==PyPI. I don't think all our users know ;)